.github/workflows/ci.yml

```diff
@@ -16,23 +16,35 @@ env:
 jobs:
   test:
     name: Test ${{ matrix.crate }}
-    runs-on: ubuntu-latest
+    runs-on: ${{ fromJSON(
+      github.repository == 'libp2p/rust-libp2p' && (
```

Comment:
This is so that if someone forks the repository and wants to run the action in their fork, it'd use the standard hosted runners (because the forks won't get access to our self-hosted infra).

Reply:
I guess that is fair, I sometimes make CI experiments on my fork before I push the branch to here.